packages/third-parties/sse/src/domain/EventStreamContext.ts

```
@@ -95,19 +95,14 @@ export class EventStreamContext {
       return this;
     }
 
-    this.responseFilter
-      .serialize(data, $ctx as any)
-      .then((data: unknown) => {
-        return this.responseFilter.transform(data, $ctx as any);
-      })
-      .then((data: unknown) => {
-        data = JSON.stringify(data);
+    this.responseFilter.transform(data, $ctx as any).then((data: unknown) => {
+      data = JSON.stringify(data);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)